File: BEE 1099.py
n = int(input())
for i in range(n):
    x, y = map(int, input().split(" "))
    if (x > y):
        temp = x
        x = y
        y = temp
    sum = 0
    for j in range(x+1, y):
        if (j % 2 != 0):
            sum += j
    print(sum)

File: BEE 1099.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
// var lines = input.split('\n');
var sep = [' ', '\n'];
// console.log(separators.join('|'));
var line = input.split(new RegExp(sep.join('|'), 'g'));

var n = parseInt(line.shift());

for (var i = 0; i < n; i++){
    var x = parseInt(line.shift());
    var y = parseInt(line.shift());
    if (x > y){
        var temp = x;
        x = y;
        y = temp;
    }
    var sum = 0;
    for (var j = x+1; j < y; j++){
        if (j % 2 !== 0){
            sum += j;
        }
    }
    console.log(sum);
}